src/main/java/com/iridium/iridiumskyblock/placeholders/Placeholders.java

```diff
@@ -176,10 +176,17 @@
 
     private static Map<String, Placeholder> getIslandTopPlaceholders() {
         HashMap<String, Placeholder> hashmap = new HashMap<>();
-        List<Island> topIslands = IridiumSkyblock.getInstance().getIslandManager().getIslands(IslandManager.SortType.VALUE);
+        
         for (int i = 1; i <= 20; i++) {
-            Optional<Island> island = Optional.ofNullable(topIslands.size() > i ? topIslands.get(i - 1) : null);
-            hashmap.putAll(getIslandPlaceholders("island_top_" + i, player -> island));
+            //To have an effectively final variable
+            final int ci = i;    
```

Comment:
You'd have to update the lines where you use this variable too
```suggestion
            final int index = i;    
```